includes/wc-product-functions.php

```diff
@@ -1048,13 +1048,17 @@ function wc_get_price_to_display( $product, $args = array() ) {
 		array(
 			'qty'   => 1,
 			'price' => $product->get_price(),
+			'price_location' => 'shop'
```

Comment:
This default name does not match the $args call below. Also for better clarity perhaps change the args name for this to display_location

Reply:
done